docs/flinkDev/building.zh.md

```diff
@@ -22,62 +22,63 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This page covers how to build Flink {{ site.version }} from sources.
+本篇主题是如何从版本 {{ site.version }} 的源码构建 Flink。
 
 * This will be replaced by the TOC
 {:toc}
 
-## Build Flink
+## 构建 Flink
 
-In order to build Flink you need the source code. Either [download the source of a release]({{ site.download_url }}) or [clone the git repository]({{ site.github_url }}).
+首先需要准备一份源码。有两种方法：第一种参考 [从发布版本下载源码]({{ site.download_url }})；第二种参考 [从 Git 库克隆 Flink 源码]({{ site.github_url }})。
 
-In addition you need **Maven 3** and a **JDK** (Java Development Kit). Flink requires **at least Java 8** to build.
+还需要准备，**Maven 3** 和 **JDK** (Java开发套件)。Flink至少依赖 **Java 8** 来进行构建。
 
-*NOTE: Maven 3.3.x can build Flink, but will not properly shade away certain dependencies. Maven 3.2.5 creates the libraries properly.
+*注意：Maven 3.3.x 可以构建 Flink，但是不会去掉指定的依赖。Maven 3.2.5 可以很好地构建对应的库文件。
 To build unit tests use Java 8u51 or above to prevent failures in unit tests that use the PowerMock runner.*
+如果运行单元，则需要 Java 8u51 以上的版本来去掉PowerMock运行器带来的单元测试失败情况。
 
-To clone from git, enter:
+从 Git 克隆代码，输入：
 
 {% highlight bash %}
 git clone {{ site.github_url }}
 {% endhighlight %}
 
-The simplest way of building Flink is by running:
+最简单的构建 Flink 的方法，执行如下命令：
 
 {% highlight bash %}
 mvn clean install -DskipTests
 {% endhighlight %}
 
-This instructs [Maven](http://maven.apache.org) (`mvn`) to first remove all existing builds (`clean`) and then create a new Flink binary (`install`).
+上面的 [Maven](http://maven.apache.org) 指令，(`mvn`) 首先删除 (`clean`) 所有存在的构建，然后构建 (`install`) 一个新的 Flink 运行包。
```

Comment:
```suggestion
上面的 [Maven](http://maven.apache.org) 指令 (`mvn`) 首先删除 (`clean`) 所有存在的构建，然后构建一个新的 Flink 运行包 (`install`)。
```

@wuchong which paren should be used? `()` or `（）`?

Reply:
@wuchong And this one ? Which one should we choose?

Reply:
Usually, we should use the Chinese characters.

Reply:
Ok, Thank you very much.